test/dtypes/test_float8_activation_int4_groupwise_preshuffle.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
should this test be in the `test/quantization/quantize_` folder?

Reply:
yeah, the test file to review is test_int4_groupwise_preshuffle.py, I forgot to remove this file